docs/requirements.txt

```diff
@@ -4,4 +4,4 @@ Pygments>=2.7.4
 m2r2==0.3.2
 sphinxcontrib-email==0.3.5
 furo
-pennylane==0.22
+pennylane==0.26
```

Comment:
Should there be a period at the top of this file?

Reply:
shouldn't cause a problem. I removed the line. We'll see if the docs build properly in main